src/userscores/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ set(MODULE_SRC
     )
 
 set(MODULE_LINK libmscore)
+set(MODULE_LINK notation)
```

Comment:
This code removes `libmscore` from MODULE_LINK and adds `notation` to MODULE_LINK, there should probably be something else ... maybe need to remove `set(MODULE_LINK libmscore)`

Reply:
removed